examples/body-to-query-params-server/app.js

```
@@ -0,0 +1,43 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+app.use(bodyParser.json())
+const axios = require('axios');
+const { json } = require('stream/consumers');
+
+const port = 3000
+
+app.post('/', (req, res) => {
+    const headerOutboundUrl = req.get('Outbound-Url');
```

This should return an error if it isn't set